content/en/monitors/service_level_objectives/error_budget.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Error Budget Monitors
+kind: documentation
+description: "Use Monitors to define the Service Level Objective"
+beta: true
+---
+
+<div class="alert alert-warning">
+This feature is in closed beta. Email <a href="mailto:slo-help@datadoghq.com">slo-help@datadoghq.com</a> to request access, to ask questions, or to provide feedback on this feature.
+</div>
+
+## Overview
+
+SLO error budget monitors are threshold based and notify you when a certain percentage of your SLO’s error budget has been consumed. 
+
+**Note:** Error budget monitors are only available for [metric-based SLOs][1].
+
+Example: Alert me if 75% of the error budget for my 7-day target is consumed. Warn me if 50% is consumed (optional).
+
+## Monitor creation
+
+1. Navigate to the [SLO status page][2].
+2. Create a new metric-based SLO or edit an existing one. For existing SLOs, you can also click the “Enable Alerts” link in the SLO detail side panel to take you directly to the alert configuration.
```

Comment:
Can we change the first sentence to: "Create a new metric-based SLO or edit an existing one, then click the ‘Save and Set Alert’ button"